schedule/migration/offline_spvm_Upgrade.yaml

```diff
@@ -20,9 +20,53 @@ schedule:
   - installation/handle_reboot
   - installation/first_boot
   - migration/post_upgrade
-  - console/system_prepare
-  - console/consoletest_setup
-  - console/zypper_lr
-  - console/check_system_info
-  - boot/grub_test_snapshot
-  - boot/snapper_rollback
+  - '{{qcow_generation}}'
```

Comment:
Just don't know that spvm already support qcow? Or just used for judgement of regression test? If so it is better to use other setting, such as REGRESSION_TEST.

Reply:
we shouldn't use spvm backend (only some Kernel tests use it and we have limited resources), we should use the hmc one (that is what we refer with PowerVM), which doesn't support snapshots.

Reply:
I have changed file name to autoyast_regression_test_offline_pre@pvm.yaml  and autoyast_regression_test_online_pre@pvm.yaml
Please check again.